src/main/java/com/hivemq/persistence/ProducerQueues.java

```diff
@@ -37,9 +37,9 @@
                                              @Nullable final SingleWriterServiceImpl.SuccessCallback<R> successCallback,
                                              @Nullable final SingleWriterServiceImpl.FailedCallback failedCallback);
 
```

Comment:
submitToAllBuckets missing

Reply:
`submitToAllBuckets` isn't currently used anywhere in the CE. Should I include it anyway?